add webin_cli_handler (#19)
EBI-Metagenomics/assembly_uploader: README.md

#### Step 2: submit the new assembly study to ENA

This step submit the XML to ENA and generate a new assembly study accession. Keep note of the newly generated study accession.
> [!NOTE]
> 

> > We recommend to submit study to TEST server first using `--test` argument. If no errors occur then re-run command **without** `--test` for live submission.

Reviewer: ```suggestion
This step submits the XML to ENA and generates a new assembly study accession identifier. Make sure to write down the newly generated study accession identifier.
> [!NOTE]
> 
> We recommend to submit a study to the TEST server first using the `--test` argument. If no errors occur, then re-run the command **without** `--test` for a live submission.
```

> > We recommend to submit study to TEST server first using `--test` argument. If no errors occur then re-run command **without** `--test` for live submission.

Reviewer 2: I would also link the ENA docs here (or just after).

Reviewer 2: Also, while you are modifing the repo. Change the title from `ENA Assembly uploader` to `ENA Metagenome Assembly uploader` or similar please

> > We recommend to submit study to TEST server first using `--test` argument. If no errors occur then re-run command **without** `--test` for live submission.

Reviewer 2: ```suggestion
> We recommend first submitting the study to ENA's TEST server using the `--test` argument. If no errors occur, then re-run the command without the `--test` option for live submission.
```

#### Step 4: upload assemblies

> Once manifest files are generated, it is necessary to use ENA's [webin-cli](https://github.com/enasequence/webin-cli) resource to upload genomes.

Reviewer 2: ```suggestion
Once the manifest files are generated, it is necessary to use ENA's [webin-cli](https://github.com/enasequence/webin-cli) resource to upload the metagenome assemblies.
```

> We recommend to use a pre-installed [**webin_cli_handler**](https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit/blob/dev/mgnify_pipelines_toolkit/ena/webin_cli_handler.py) script.

Reviewer: ```suggestion
We recommend using a pre-installed [**webin_cli_handler**](https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit/blob/dev/mgnify_pipelines_toolkit/ena/webin_cli_handler.py) script.
```
isn't this shipping with the software? What do you mean pre-installed?

Author: That tool was already included as dependency that is what I meant it 'pre-installed'

> [!NOTE]
> 

> > First, validate your submission with `--mode validate`. \

Reviewer 2: ```suggestion
> First, validate your submission with the `--mode validate` option. \
```

> > Second, upload to TEST server using `--test` (make sure you have submitted a study to TEST server on Step 2). 

Reviewer: ```suggestion
> Second, upload to the TEST server using `--test` (make sure you have submitted a study to the TEST server on Step 2). 
```

> > Second, upload to TEST server using `--test` (make sure you have submitted a study to TEST server on Step 2). 

Reviewer 2: ```suggestion
> Second, upload to ENA's TEST server using the `--test` flag (make sure you have submitted the study to ENA's TEST server on Step 2). 
```

Run live execution:

```bash
webin_cli_handler \
  --manifest *.manifest \
  --context genome \
  --mode submit \
  [--test]
```

> If you do not have ena-webin-cli installed add `--download-webin-cli`. \

Reviewer 2: ```suggestion
If you do not have ena-webin-cli installed add the `--download-webin-cli` flag, the tool will be autoamtically downloaded. It requires a recent JAVA version to be able to work, [official repo](https://github.com/enasequence/webin-cli). \
```
